deployer/scripts/hawkular.sh

```diff
@@ -121,34 +57,10 @@ EOF
   oc process hawkular-cassandra-services | oc create -f -
   oc process hawkular-support | oc create -f -
 
-  echo "Creating the Hawkular Metrics Route"
-  # We need to create the route after the service has been created so that its labels get applied to the route itself
-  route_params="--hostname=$hawkular_metrics_hostname --service=hawkular-metrics --dest-ca-cert=$dir/hawkular-metrics-ca.cert"
-  if [ -s ${secret_dir}/hawkular-metrics.pem ]; then
-    `openssl rsa -in  ${secret_dir}/hawkular-metrics.pem > $dir/custom-certificate.key`
-    # We want to get all the certificates in the pem which is a bit tricky. The more simple 'openssl x509 ...' command will not work since it only returns the first certificate
-    `openssl crl2pkcs7 -nocrl -certfile ${secret_dir}/hawkular-metrics.pem | openssl pkcs7 -print_certs | grep -v "^subject=*\|^issuer=*\|^$" > $dir/custom-certificate.crt`
-     route_params="${route_params} --cert=$dir/custom-certificate.crt --key=$dir/custom-certificate.key"
-     if [ -s ${secret_dir}/hawkular-metrics-ca.cert ]; then
-       route_params="${route_params} --ca-cert=${secret_dir}/hawkular-metrics-ca.cert"
-     fi
-  fi
   # this may return an error code if the route already exists, this is to be expect with a refresh and is why we have the || true here
-  oc create route reencrypt hawkular-metrics ${route_params} || true
+  ## once BZ 1401081 is done, the Route specified on `hawkular-metrics.yaml` should work and this command here should be removed.
+  oc create route passthrough --service hawkular-metrics || true
 
-  # if we are dealing with a refresh, then we need to update the router's CACertificate for the Hawkular Metric's internal certificate
-  if [ "$mode" = "refresh" ]; then
-   type=$(oc get route hawkular-metrics --template='{{.spec.tls.termination}}') || true
-   if [ "$type" = "reencrypt" ]; then
-      oc patch route hawkular-metrics -p "$(python <<-EOF
-				import json
-				ca = open("$dir/hawkular-metrics-ca.cert").read()
-				print json.dumps({"spec": {"tls": {"destinationCACertificate": ca}}})
-				EOF
-      )"
-   fi
-  fi
- 
```

Comment:
A cert is created once via a `secret`. Similarly, a `service-ca` is created once for the lifecycle of a cluster. So, even if the cert's secret is removed, it will still be signed by the same CA.